packages/x-charts/src/BarChart/seriesConfig/tooltip.ts

```diff
@@ -10,6 +10,11 @@ const tooltipGetter: TooltipGetter<'bar'> = (params) => {
 
   const label = getLabel(series.label, 'tooltip');
   const value = series.data[identifier.dataIndex];
+
+  if (!value) {
+    return null;
+  }
+
```

Comment:
```suggestion

  if (value === undefined) {
    return null;
  }

```

For now I agree it's the same because it's impossible to trigger the tooltip with a bar element with value `0` or `null` because you can not hover those elements.

But as soon as we introduce the control on tooltip. If `tooltipItem={{ type; 'bar'; id, dataIndex }}` match a bar element with value 0, we will have to display it